config/grunt/custom-tasks/init-rc-release.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+/**
+ * helper function
+ * @param {array} 
+ * @returns {bool}
+ */
+
+if (!String.prototype.isInList) {
+	String.prototype.isInList = function() {
+	   let value = this.valueOf();
+	   for (let i = 0, l = arguments.length; i < l; i += 1) {
+		  if (arguments[i] === value) return true;
+	   }
+	   return false;
+	}
+ }
+
+
+
+
+
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"init-rc-release",
+		"central entry point to set options in one place",
+		function() {
+            const options = this.options({
+                failOnError: true,
+                enableSvnDeploy: false,
+                githubOwner: '',
+                githubRepo: '',
+                alternativeBranch: 'CI-test',
```

Comment:
This variable is `ci-rc-test` somewhere else, should this be the same value?

https://github.com/Yoast/wordpress-seo/pull/15910/files#diff-4513b0c522f70242f8e8e0fd1517c730R17

Reply:
true